roottest/root/io/compression/CMakeLists.txt

```diff
@@ -1,7 +1,47 @@
-#-------------------------------------------------------------------------------
-#
-# Placeholder file to translate the tests to the new CTest system. Meanwhile we
-# define a CTest test that calls 'make' in ${CMAKE_CURRENT_SOURCE_DIR}
-#
-#-------------------------------------------------------------------------------
-ROOTTEST_ADD_OLDTEST()
+ROOT_GENERATE_DICTIONARY(G__IoCompressionGeneration
+  ${CMAKE_CURRENT_SOURCE_DIR}/Event.h
+  LINKDEF ${CMAKE_CURRENT_SOURCE_DIR}/EventLinkDef.h)
+
+ROOT_LINKER_LIBRARY(IoCompressionGeneration
+  ${CMAKE_CURRENT_SOURCE_DIR}/IoCompressionGeneration.cxx
+  G__IoCompressionGeneration.cxx
+  LIBRARIES Core Tree Hist MathCore Physics Graf Matrix)
+
+if(MSVC)
+  add_custom_command(TARGET IoCompressionGeneration POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/libIoCompressionGeneration.dll
+                                     ${CMAKE_CURRENT_BINARY_DIR}/libIoCompressionGeneration.dll
+    COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_BINARY_DIR}/$<CONFIG>/libIoCompressionGeneration.lib
+                                     ${CMAKE_CURRENT_BINARY_DIR}/libIoCompressionGeneration.lib)
+endif()
+
+ROOTTEST_GENERATE_EXECUTABLE(IoCompressionGenerator
+  ${CMAKE_CURRENT_SOURCE_DIR}/IoCompressionGenerator.cxx
+  LIBRARIES Core RIO Net Tree Hist MathCore IoCompressionGeneration
+  DEPENDS IoCompressionGeneration
```

Comment:
From https://github.com/root-project/roottest/pull/774#discussion_r2020253039:

This actually needs to be a `FIXTURES_REQUIRED` rather than a `DEPENDS.`